docs/sources/shared/visualizations/axis-options-all.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Axis options
+comments: |
+  This file is used in the following visualizations: time series.
+---
+
+Options under the **Axis** section control how the x- and y-axes are rendered. Some options don't take effect until you click outside of the field option box you're editing. You can also press `Enter`.
+
+| Option                             | Description                                                                                                        |
+| ---------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| Time zone                          | Set the desired time zones to display along the x-axis.                                                            |
+| [Placement](#placement)            | Select the placement of the y-axis.                                                                                |
```

Comment:
Should this definition change to align with the one for state timeline and status history or no?

Reply:
Its more fully defined at the anchor so nah, we can leave this as is imo